generators/typescript-v2/ast/src/ast/TypeLiteral.ts

```diff
@@ -43,6 +48,10 @@ interface Tuple {
     values: TypeLiteral[];
 }
 
+interface Nop {
```

Comment:
What's `Nop` here? Should it be `NoOp`?

Reply:
https://github.com/fern-api/fern/pull/5435#discussion_r1890671038